docs/docs/components/guidelines/README.md

```diff
@@ -552,4 +762,823 @@ end user. Generate and use a GUID for the shared secret value, save it to a
 
 ### Action Guidelines
 
-_(Coming soon)_
\ No newline at end of file
+_(Coming soon)_
+
+## Reviewing Code
+
+### Preliminary Steps
+
+As a code reviewer, whenever you'd like to review a pull request (PR) there's a
+few preliminary steps that we'd like you to take in order to avoid
+misunderstandings, wasted time and effort:
+
+1. Targeting a PR:
+   1. Check the [main Kanban board](/code/projects/1) for any tasks pending
+      review (i.e. the **Ready for PR review** column). These tasks should be
+      associated to an open PR that is waiting for a review.
+   2. If the above step does not yield any PR, explicitly look for open pull
+      requests in the [pull requests page](/code/pulls) without reviews and
+      without reviewers assigned to it. [This
```

Comment:
Maybe remove the final sentence here and make "open pull requests without reviews and without reviewers assigned to it" the link.

Reply:
You mean remove this sentence?

> [This link](/code/pulls?q=is%3Apr+is%3Aopen+review%3Anone+no%3Aassignee) already provides a filtered view for the mentioned criteria.